specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -0,0 +1,469 @@
+          "readOnly": true
+        },
+        "instanceName": {
+          "description": "The name of the resource instance that the usage is about.",
+          "type": "string",
+          "readOnly": true
+        },
+        "instanceId": {
+          "description": "The uri of the resource instance that the usage is about.",
+          "type": "string",
+          "readOnly": true
+        },
+        "instanceLocation": {
+          "description": "The location of the resource instance that the usage is about.",
+          "type": "string",
+          "readOnly": true
+        },
+        "currency": {
+          "description": "The ISO currency in which the meter is charged, for example, USD.",
+          "type": "string",
+          "readOnly": true
+        },
+        "usageQuantity": {
+          "description": "The quantity of usage.",
+          "type": "number",
+          "format": "decimal",
+          "readOnly": true
+        },
+        "billableQuantity": {
+          "description": "The billable usage quantity.",
+          "type": "number",
+          "format": "decimal",
+          "readOnly": true
+        },
+        "pretaxCost": {
+          "description": "The amount of cost before tax.",
+          "type": "number",
+          "format": "decimal",
+          "readOnly": true
+        },
+        "isEstimated": {
+          "description": "The estimated usage is subject to change.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "meterId": {
+          "description": "The meter id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "meterDetails": {
+          "description": "The details about the meter. By default this is not populated, unless it's specified in $expand.",
+          "$ref": "#/definitions/MeterDetails",
+          "readOnly": true
+        },
+        "subscriptionGuid": {
+          "description": "Subscription guid.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionName": {
+          "description": "Subscription name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "accountName": {
+          "description": "Account name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "departmentName": {
+          "description": "Department name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "product": {
+          "description": "Product name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "consumedService": {
+          "description": "Consumed service name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "costCenter": {
+          "description": "The cost center of this department if it is a department and a costcenter exists",
+          "type": "string",
+          "readOnly": true
+        },
+        "additionalProperties": {
```

Comment:
Is this supposed to be a container of additional data?

Reply:
yes but not exclusively as a container. Additional properties are something that users can also key in things on a resource in Azure and this is where it shows up.
However I am not sure what are varieties here since the downstream data gets populated through a lot of teams in Azure here :)

Reply:
I am sorry I didn't look at your edited comment.
Ok yes, that's a miss on my part, I should have updated the examples to reflect the "string" type. Will do now. 
Btw, how do I look at this Travis job? The one below says everything is passed

Reply:
Click on the job and look for the items marked with `PR_ONLY=true` (which means it's testing stuff in your PR) -- soon, the validator will actually block the PR for those, at the moment, we have to manually check that .

Reply:
Got it, Thanks. Pushed the correction too